src/properties-mapping.js

```diff
@@ -28,14 +28,14 @@ const propertyMapper = {
   streamedFileOffset: setPropertyMapper('seekTo', 'streamedFileOffset'),
   // Model
   modelPath: setPropertyMapper('path', 'modelPath'),
-  modelResourceId: setPropertyMapper('resourceId', 'modelResourceId'),
+  modelResourceId: setPropertyMapper('resourceId', ''),
```

Comment:
Why is the old property name removed ?

Reply:
I found the reason: 
The property mapper attempts to read the property value using the old name if it fails the mapper will use the default value, which is provided by getPropertyValue(...) using the new name.
Is this correct ?